internal/pkg/manager/spod/setup.go

```diff
@@ -123,6 +124,16 @@ func getEffectiveSPOd(dt *daemonTunables) *appsv1.DaemonSet {
 
 	sepolImage := &refSPOd.Spec.Template.Spec.InitContainers[1]
 	sepolImage.Image = dt.selinuxdImage // selinuxd ships the policies as well
+
+	// Disable SELinux configuration when SELinux is not in enforcing mode
+	if selinux.EnforceMode() != selinux.Enforcing {
+		for i := range refSPOd.Spec.Template.Spec.InitContainers {
+			refSPOd.Spec.Template.Spec.InitContainers[i].SecurityContext.SELinuxOptions = nil
+		}
+		for i := range refSPOd.Spec.Template.Spec.Containers {
+			refSPOd.Spec.Template.Spec.Containers[i].SecurityContext.SELinuxOptions = nil
+		}
+	}
```

Comment:
I don't think we should be doing this, instead you could just turn off SELinux in the SPO deployment.